powerauth-push-server/src/main/java/io/getlime/push/controller/rest/DeviceController.java

```diff
@@ -80,7 +80,7 @@ void setClient(PowerAuthServiceClient client) {
             registration.setAppId(appId);
             registration.setPushToken(pushToken);
         }
-        registration.setLastRegistered(new Date());
+        registration.setTimestampCreated(new Date());
```

Comment:
The same issue with SQL. It is not our goal to have all timestamps called "timestamp created" - we need to name the timestamp based on what the timestamp represents. This one is timestamp of last device registration and should be named appropriately...